pdf-converter/config/config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "secret": "e32bd326eae57036be5c17f1bd6b9bd36b7e5b8e",

```

Comment:
Is this supposed to be hardcoded or replaced by the environment variable?
@pradyu1

Reply:
This is removed, we are now reading variables from the environment